src/tools/wasm-reduce.cpp

```diff
@@ -414,8 +414,9 @@ struct Reducer : public WalkerPass<PostWalker<Reducer, UnifiedExpressionVisitor<
       functionNames.push_back(func->name);
     }
     size_t skip = 1;
+    bool winning = false;
```

Comment:
Nit: should be `didRemove` because no fun allowed 😛

Reply:
Heh, fair enough. How about "justRemoved"? (since it's about the last iteration, I think "just" captures that better. also added a comment)

Reply:
Yeah that captures the nuance there better. The comment makes it perfect.